submissions/AsyaYeromina/Tiny-JS-world/index.js

```diff
@@ -0,0 +1,99 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/AsyaYeromina/a-tiny-JS-world
+   Web app: https://asyayeromina.github.io/a-tiny-JS-world/.
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const dog = {
+   species: 'Canis',
+   name: 'Jessie',
+   gender: 'female',
+   legs: '4',
+   hands: '0',
+   speech: 'woof-woof!',
+   friends: ['Stepan', 'Ivanka']
+};
+
+const cat = {
+   species: 'Felis catus',
+   name: 'Python',
+   gender: 'male',
+   legs: '4',
+   hands: '0',
+   speech: 'Meow',
+   friends: []
+};
+
+const woman = {
+   species: 'Homo sapiens',
+   name: 'Ivanka',
+   gender: 'female',
+   legs: '2',
+   hands: '2',
+   speech: 'Hello!',
+   friends: ['Stepan', 'Python', 'Jessie']
+};
+
+const man = {
+   species: 'Homo sapiens',
+   name: 'Stepan',
+   gender: 'male',
+   legs: '2',
+   hands: '2',
+   speech: 'Hello!',
+   friends: ['Ivanka', 'Python', 'Jessie']
+};
+
+const catWoman = {
+   species: 'Felis sapiens',
+   name: 'Selina',
+   gender: 'female',
+   legs: '2',
+   hands: '2',
+   speech: cat.speech,
+   friends: ['Stepan', 'Python']
+};
+
+const inhabitants = [cat, dog, woman, man, catWoman];
+const keys = ['species', 'name', 'gender', 'legs', 'hands', 'speech', 'friends'];
+
+const getValues = function(obj, arr) {
+   const stringArr = [];
+   arr.forEach(key => {
+      const value = obj[key];
+      if (typeof value === 'object') {
```

Comment:
This is an issue. If the value is not an iterable object (like literal objects above), then `value.join()` will throw an error.
The proper way to check if an entity is an array is to employ `Array.isArray(variable)`.

P.S. No need to handle cases where a value is a non-iterable object for now.

Reply:
I thought that `Array.isArray(variable)` can't be used in this home task
This is what I need. Added to code. Thanks

Reply:
@AsyaYeromina what made you think so?

Reply:
At that time I didn't understand which methods can be used, and which shouldn't and why. 
It's a surprise for me now, that I couldn't comprehend this before, because the task description seems very clear now, and the job seems to me now very easy to complete.